imports/api/custom-users.js

```diff
@@ -148,5 +149,18 @@ Meteor.methods({
         [`notificationSettings.${settingName}`]: !!isOn
       }
     })
+  },
+  'resendEmail': function (userId) {
+    if (Meteor.isServer) {
+      try {
+        check(userId, String)
+        return Accounts.sendVerificationEmail(userId)
+      } catch (e) {
+        let userId = Meteor.userId()
```

Comment:
It should get the userId as arg, and then if it fails get it from Meteor.userId().
It should just use Meteor.userId() always, and not expect any args